server/services/service.ts

```diff
@@ -37,15 +37,14 @@ export default ({ strapi }: { strapi: Strapi }) => ({
   isEncrypted(value: string): boolean {
     if (!value) return false;
 
-    const textParts = value.split(':');
-    const firstPart = textParts.shift();
+    const dotsPos = value.indexOf(':');
 
-    if (!firstPart) throw new Error('Malformed payload');
+    if (dotsPos < 0) return false
 
-    const iv = Buffer.from(firstPart, 'hex');
+    const iv = Buffer.from(value.slice(0, dotsPos), 'hex');
 
     // Test first indications of encrypted or not
-    if (iv.length !== IV_LENGTH && !/[0-9A-Fa-f]{6}/g.test(value)) {
```

Comment:
The test is redundant because `Buffer.from(firstPart, 'hex');` filters out invalid hexadecimal, so if the test fails the IV length would have been 0 anyways